tests/acceptance/edit-note-test.js

```diff
@@ -0,0 +1,33 @@
+import { test } from 'qunit';
+import moduleForAcceptance from 'remember/tests/helpers/module-for-acceptance'
+
+moduleForAcceptance('Acceptance | edit note')
+
+  test('should edit a note when user clicks edit this reminder', function(assert) {
+    visit('reminders/new');
```

Comment:
In the future you can use Mirage to create a record for you so you dont have to retest that you can add a reminder. `server.createList('reminder', 1)` then Mirage will do the heavy lifting for you